modules/delegates/hooks/useDelegateCreate.ts

```diff
@@ -35,7 +38,15 @@ export const useDelegateCreate = (): LockResponse => {
   );
 
   const create = callbacks => {
-    const freeTxCreator = () => voteDelegateFactory.create();
+    if (!account || !provider) {
+      return;
+    }
+
+    const freeTxCreator = async () => {
```

Comment:
I know you didn't name this function but I'm seeing it should actually be called `createTxCreator` instead of `lockTxCreator`. Might as well update in this PR.

Also noticing we define `LockResponse` as a type in many of these hooks files. We should probably give that a better name and export it from somewhere. Again, not something you wrote in this PR but maybe a good opportunity to update it.